src/applications/pensions/config/chapters/04-household-information/marriageHistory.js

```diff
@@ -27,6 +28,12 @@ const dateSchema = {
 
 /** @type {PageSchema} */
 export default {
+  title: (form, { pagePerItemIndex } = { pagePerItemIndex: 0 }) =>
+    getMarriageTitleWithCurrent(form, pagePerItemIndex),
+  path: 'household/marriages/:index',
+  depends: isMarried,
```

Comment:
I have mixed feelings on moving the `depends` to the page files - like the title and path, they relate strongly to the overall form flow